web/src/assets/styles/blocks.scss

```diff
@@ -207,6 +207,6 @@ section > .content {
   margin-left: 4px;
 }
 
-.has_help {
+.with_description {
   cursor: pointer;
 }
```

Comment:
Ok, let me clarify the situation for future readers:

My previous comment was motivated by a miss understanding from my side. After talking about it in IRC I realized that you are using this CSS *utility* class for the element that triggers the `Popover`, not for it contents itself. My fault, please excuse me. That's one thing I sometimes _hate_ from PatternFly: sometimes the content is the children, sometimes is a prop. I'm pretty sure they have strong reasons for that... but I'd use the children as content and the prop for creating the trigger :)

So far, so good. It makes sense that show the right cursor to the users in order to make them easy to detect that there is something clickable. Thus, I think we should use a native clickable element. As we discussed in IRC too, GitHub uses a very nice and well styled `<details>` for building their own popovers. Very clever, IMHO. But please, go ahead with your approach for this PBI. We can re-think that thing later... and I personally think you deserve both, to get the PR merged and to work in an improvement, if needed, later.

Regarding the CSS utility class, I'd do two things:

* Name it `cursor-pointer`, which makes it more explicit and generic.
* Place it in the `utilities.scss` file instead. 

I really hope to not see an intensive use of this CSS class across the project... because it does not make sense at all. 

Thanks for the feedback gave in the IRC Libera #yast channel and, again, excuse me for the noise :) In any case, I feel it has been constructive in the end.